docs/source/technical_tutorials/search/cmr_search_in_r.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>This isn't a particularly R way of going about it, and results in a nested list of lists (at least in the print statement), fyi</p><p><code>print(link_list[1:5])</code> would have also worked. This issue currently is that to get the url you have to do</p><p><code>link_list[[1]]</code> instead of <code>link_list[1]</code>   </p><p> </p><p>Rather you can replace the whole cell with:</p>

<pre class="ql-syntax" spellcheck="false">s3_urls &lt;- sapply(results, function(x){x['Granule']['OnlineAccessURLs'][[1]][1]['URL']})

print(s3_urls[1:5])

## http urls
https_urls &lt;- sapply(results, function(x){x['Granule']['OnlineAccessURLs'][[1]][0]['URL']})
</pre><p>Alternately we could modify the loop, I don't think you actually want a List, but rather a Vector in R</p>

<pre class="ql-syntax" spellcheck="false">link_list &lt;- c()

for (result in results) {
&nbsp;&nbsp;&nbsp; link_list &lt;- c(link_list, (result['Granule']['OnlineAccessURLs'][[1]][1]['URL']))
}

print(link_list[1:5])
</pre><p>would get the same answer as the sapply method but maybe be easier for most people to understand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>